Mathlib/RingTheory/Localization/FieldRange.lean

```diff
@@ -0,0 +1,85 @@
+/-
+Copyright (c) 2024 Jz Pan. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jz Pan
+-/
+import Mathlib.RingTheory.Localization.FractionRing
+import Mathlib.FieldTheory.Adjoin
+
+/-!
+
+# Some results on image of a ring and its fraction field under ring homomorphisms
+
+-/
+
+namespace IsFractionRing
+
+variable {F A K L : Type*} [Field F] [CommRing A] [Algebra F A]
+  [Field K] [Algebra F K] [Algebra A K] [IsFractionRing A K] [Field L] [Algebra F L]
+
+variable (A K) in
+/-- If `A` is a ring with fraction field `K`, then the subfield of `K` generated by the image of
+`algebraMap A K` is equal to the whole field `K`. -/
+theorem closure_range_algebraMap : Subfield.closure (Set.range (algebraMap A K)) = ⊤ :=
```

Comment:
Putting it in this file means people will need ~30 more transitive imports to use it. But not adding imports (even just 2) to an existing file is also nice ...

Reply:
> Putting it in this file means people will need ~30 more transitive imports to use it. But not adding imports (even just 2) to an existing file is also nice ...

What's your suggestion on the file? It seems that `Subfield` is lightweight, while `IntermediateField` is heavy... So it's possible to move 3 results on `Subfield` to earlier files, by adding a little bit more imports to that file.

In application I need to use `IntermediateField` version, though.

Reply:
I found that `Mathlib.FieldTheory.Adjoin` already imports `IsFractionRing`. So these `AlgHom` version can be moved to that file 😂 and we don't need to introduce a new file anymore. Do you think it's a good idea?

Reply:
Yeah, let's move the remaining two results to FieldTheory/Adjoin. It's natural since they are about `IntermediateField.adjoin`.

Reply:
Done.